src/app/services/code_service.js

```diff
@@ -49,7 +49,16 @@ angular
             query.push(line);
         });
 
-        var rel = [model.database, model.schema, model.identifier || model.alias || model.name].join(".");
+        var database;
+        if (!model.database) { 
+            database = '' 
+        } else {
+            database = model.database + '.'
+        }
+
+        var rel;
+        rel = database + model.schema + "." + model.identifier;
```

Comment:
Nonblocking, but this can be cleaned up a bit:
```suggestion
        const database = model.database ? model.database + '.' : '';
        const rel = database + model.schema + "." + model.identifier;
```

Reply:
I was wondering if it should be written in a more "javascript" way or following the existing code base 😄